cluster/gce/configure-vm.sh

```diff
@@ -386,7 +378,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: '$(echo "$MINION_IP_RANGE" | sed -e "s/'/''/g")'
+  cbr-cidr: 10.123.45.0/30
```

Comment:
I think both kubelet and kube-proxy are going to use host network even they are running as container. I am not concerning about kubelet and kube-proxy. But I do concern about master components. Today we are running them in a pod by using host networking, but in the future for HA master, we cannot use host networking for them. In this case, we might allocate ip for to-be-deprecated ip range here. But this doesn't break today's model, and we can iterate this latter, I think.